src/tools/collect-license-metadata/src/reuse.rs

```diff
@@ -34,7 +34,14 @@ pub(crate) fn collect(
 
 fn obtain_spdx_document(reuse_exe: &Path) -> Result<String, Error> {
     let output = Command::new(reuse_exe)
-        .args(&["--include-submodules", "spdx", "--add-license-concluded", "--creator-person=bors"])
+        // Ferrocene note: changed to assume reuse is installed via "uv" instead of directly
+        .args(&[
+            "reuse",
+            "--include-submodules",
+            "spdx",
+            "--add-license-concluded",
+            "--creator-person=bors",
+        ])
```

Comment:
We should not do this. The reason why `reuse` is searched on the filesystem and can be overridden in `config.toml` is that customers who build Ferrocene from source might do it from an airgapped system where uvx doesn't work. We should instead [install REUSE as a tool in CI](https://docs.astral.sh/uv/guides/tools/#installing-tools).